### ISSUE TITLE
Having trouble reproducing the reported accuracy

### ISSUE THREAD

**Issue by wishforgood** [mrastegari/XNOR-Net#2]
I ran the code with command `th main.lua -data ./images -nGPU 2 -batchSize 512 -netType alexnet -binaryWeight -dropout 0.1` after changed the learning rate policy to be 

```
1, 4, 1e-1, 5e-4,
5, 8, 1e-3, 5e-4,
9, 12, 1e-5, 0.
13, 16, 1e-7, 0
```

I tried to use this to get the same result for BWN(alexnet) as reported in the paper. However, the resulting top-1 train accurcy after the first epoch is 7.82%, far from reported. The top-5 training accuracy is 19.72%. Is there anything I missed?

**Comment by mrastegari** [comment]
Have you tried to load the pretrained model to check that if you can get the same accuracy? 
try with 1 GPU batchSize 128 with the default LR regime. epochNumber 10000. I have noticed some times multiGPU has problem in aggregating the gradients in the backward pass in different machines.

**Comment by wishforgood** [comment]
Thanks for your answer! Yes, I've checked the pretrained model and it works, I got the same accuracy. I've also tried 1 GPU with batchSize 128 with the default LR regime, the accuracy is increasing. I will train it further to get the accuracy curve, but is there any curve I  can depend on? It would be better if I can check if the training is working in the early epoches of training.

**Comment by mrastegari** [comment]
I will try to create the learning curve and share it.

**Comment by wishforgood** [comment]
OK, please inform me at that time.

**Comment by mrastegari** [comment]
please see the discussion here https://github.com/mrastegari/XNOR-Net/issues/3

**Comment by wishforgood** [comment]
I have finished 55 epoches and found that the resulting top-1 accuracy was only 32.49%, the top-5 accuracy was only 57.08%. So there must be something wrong. I have deleted your training cache and let the program to produce a new training cache itself because the subfolders in training set of mine was renamed in another manner(0001, 0002, 0003.... different from the official manner). Will it be the reason of not reproducing your result?
